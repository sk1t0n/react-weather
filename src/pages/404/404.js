import Alert from 'react-bootstrap/Alert';

const NotFoundPage = (props) => (
  <Alert className="max-width-600_center" variant="danger">404 - Not found</Alert>
);

export { NotFoundPage };
