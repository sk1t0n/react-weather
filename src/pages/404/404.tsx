import React from 'react';
import Alert from 'react-bootstrap/Alert';

export const NotFoundPage: React.FC = () => (
  <Alert className="max-width-600_center" variant="danger">404 - Not found</Alert>
);
