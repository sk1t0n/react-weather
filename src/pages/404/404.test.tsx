import { render } from '@testing-library/react';
import { NotFoundPage } from './404';

describe('NotFoundPage component', () => {
  it('check the snapshot', () => {
    const { asFragment } = render(<NotFoundPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
