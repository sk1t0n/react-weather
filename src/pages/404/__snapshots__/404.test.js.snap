// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component check the snapshot 1`] = `
<DocumentFragment>
  <div
    class="fade max-width-600_center alert alert-danger show"
    role="alert"
  >
    404 - Not found
  </div>
</DocumentFragment>
`;
